tests/integration/incremental_unique_id_test/models/trinary_unique_key_list.sql

```diff
@@ -0,0 +1,19 @@
+-- types needed to compare against expected model reliably
```

Comment:
They are used in different test cases, `test_trinary_unique_key_list` and `test_trinary_unique_key_list_no_update`.
Shall we leave them to keep the tests from going too far away from the upstream tests; otherwise it could be more difficult to maintain when the upstream tests are updated.

Reply:
Sounds good. I am wondering if these model files can be reused across different test cases or it's a better practice to not reuse them?

Reply:
:+1: Although I still didn't get what does the comment mean ("types needed to compare against expected model reliably" and "for comparing against auto-typed seeds")

Reply:
Yeah, we also need to keep an eye on dbt-spark PRs.